apps/hash-api/src/index.ts

```diff
@@ -294,6 +284,60 @@ const main = async () => {
     return jsonParser(req, res, next);
   });
 
+  /**
+   * Proxy for requests to the Ory Kratos public API, to be consumed by the frontend.
+   *
+   * Although the proxy would ideally come before the body parser, so that we're passing it through untouched,
+   * we check the body in this process in order to rate limit requests based on the user attempting to log in.
+   */
+  app.use(
+    "/auth/*",
+    authRouteRateLimiter,
+    userIdentifierRateLimiter,
+    cors(CORS_CONFIG),
+    (req, res, next) => {
```

Comment:
I assume this would mean we are limiting any kratos request to 5 per 20 seconds, and not rate limiting per kratos route. I can't think of a reason this would be an issue however.

Also not sure we should be rate limiting the `/whoami` request, but we are no longer calling this on every page load so I guess it shouldn't be a problem either.